app/main.py

```diff
@@ -16,4 +16,10 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [my_favourite_films,
+                                marks, collection_of_coins],
+                    "immutable": [
+                        lucky_number, pi,
+                        one_is_a_prime_number,
+                        name, profile_info]
+                    }
```

Comment:
Please, make sure that there is no code on the same line with `{` and place each variable on a new line so that there would be no variables on the same line with `[` and `]` as well